rclpy/rclpy/impl/rmw_implementation_tools.py

```diff
@@ -85,5 +88,4 @@ def import_rmw_implementation():
     module_name = '._rclpy__{rmw_implementation}'.format(
         rmw_implementation=__selected_rmw_implementation,
     )
-    __rmw_implementation_module = importlib.import_module(module_name, package='rclpy')
```

Comment:
ah, you guys are too fast (and I have not been paying close enough attention - sorry for wasting your time on this.)